### PR TITLE
CLUS-firepower-restapi109-md1

### PR DESCRIPTION
**Pull request by ReneeBornstein**
* removed capitals from headings
* added parts of sentences
* changed a long sentence to a bullet
   list
* added commas and colons

### CODE REVIEW

File: labs/firepower-restapi-109/1.md [hunk 1]
```diff
@@ -1,33 +1,36 @@
 # FMC - REST - API - 109
 
-## Lab Objective
+## Lab objective
```
Comment (annegentle) [inline labs/firepower-restapi-109/1.md]: The Lab and Learning Objective sections can be combined into ## Objectives.

File: labs/firepower-restapi-109/1.md [hunk 2]
```diff
@@ -53,12 +56,14 @@ Here is the snapshot illustrating the above:
...
-Once you create an AC policy successfully using the previous step, look for the AC Policy ID in the RESPONSE TEXT of the POST call
+
+Once you create an AC policy successfully using the previous steps, look for the AC Policy ID in the RESPONSE TEXT of the POST call.
 
 Here is the snapshot illustrating the above:
 ![](/posts/files/firepower-restapi-109/assets/images/expl-02.png)
 
-*Look for ID( "id" )* in the response window of API explorer
+
+*Look for ID( "id" )* in the response window of API explorer:
```
Comment (annegentle) [inline labs/firepower-restapi-109/1.md]: I think it was API Explorer previously.

**Review by annegentle — CHANGES_REQUESTED** [review]

**Review by annegentle — APPROVED** [review]

**Comment by annegentle** [comment]
I'll merge this now but will still need the /post/files/ references cleaned up.

**Comment by ReneeBornstein** [comment]
Sorry, what does /post/files mean?

**Pull request merged**